### PR TITLE
Initial structure updated

### PR DESCRIPTION
**Pull request by Akons97**
I accidentally created one more branch for the same issue...

### CODE REVIEW

File: report/REPORT.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+*TODO - dependencies and important interactions of subsystems*
+
+
+
+## State of solution
+*TODO - current state of our system, results of static analysis and code quality assessment*
+
+
+
+## Choices
```
Comment (C0DK) [inline report/REPORT.md:23]: I thought we agree to split this into the relevant themes during our meeting. I.e use the document as a basis for both `application solution`, `CI/CD`, and maybe a seperate chapther that is a combination of logging and monitoring.. I might be wrong though.

File: report/REPORT.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+*TODO - argue for the choice of technologies and decisions. We can use CHOICES.md here.*
+
+
+## Process' details
```
Comment (C0DK) [inline report/REPORT.md:27]: What is the difference between this and development practices? :D afaik one is a subsection of the other.

File: report/REPORT.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+## Development practices
+*TODO - organization of our repository, branching strategy, process and tools (how do we use issues, tasks, kanban board)*
+
+
+## Monitoring
+*TODO - how(and what) do we monitor in our system*
+
+
+## Logging
```
Comment (C0DK) [inline report/REPORT.md:43]: maybe combine with monitoring

File: report/REPORT.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+*TODO - what do we log and how we aggregate it*
+
+
+## Security assessment
```
Comment (C0DK) [inline report/REPORT.md:47]: Someone mentioned that we probably shouldn't have our whole postmortem, in the report as it is a separate hand-in - i'm not sure if this is not the case though. 

However it seems more like a subsection of `state of solution` imo - However i'd love to hear opinions of everyone else.

File: report/REPORT.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+*TODO - brief security assessment(we can use POSTMORTEM.md)*
+
+
+## Scaling and load balancing
```
Comment (C0DK) [inline report/REPORT.md:51]: Maybe we should have a section that is a bit `misc` called `Devops tools` where scaling, loadbalancing, logging, monitoring are all subsections?

File: report/REPORT.md (hunk 4 above)
Comment (Akons97) [reply report/REPORT.md:47]: Not the whole postmortem, just a short excerpt

File: report/REPORT.md (hunk 1 above)
Comment (Akons97) [reply report/REPORT.md:23]: You're right, I'll remove Choices so we'll split that document into all subsections

File: report/REPORT.md (hunk 2 above)
Comment (Akons97) [reply report/REPORT.md:27]: It's basically the same thing

File: report/REPORT.md (hunk 3 above)
Comment (Akons97) [reply report/REPORT.md:43]: I agree since those 2 sections won't be long. Better to have 1 instead

File: report/REPORT.md (hunk 5 above)
Comment (Akons97) [reply report/REPORT.md:51]: I like the idea! Will do that

**Pull request merged**